x-pack/plugins/security_solution/scripts/endpoint/common/roles_users/index.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import * as withResponseActionsRole from './with_response_actions_role.json';
```

Comment:
nit: you can directly export this so you don't need the export below:
```typescript
export * as withResponseActionsRole from './with_response_actions_role.json';
```

Same with the others

Reply:
I'm not sure if this is a better way to do this, but in order to import this json object without a `default` key I need to first import it as a named import and then export it separately. We also have some lint rule where we can't export `*` directly.